spec/features/social_spec.rb

```diff
@@ -21,26 +21,20 @@
       expect_page_to_contain_meta_tag("keywords", "first_keyword, second_keyword")
     end
 
-    scenario 'should have sherable buttons' do
-      expect(page).to have_css(".social-share-button-twitter")
-      expect(page).to have_css(".social-share-button-facebook")
-      expect(page).to have_css(".social-share-button-google_plus")
-    end
-
     scenario 'should have open graph meta tags' do
       expect(page).to have_css("meta[property='og:title'][content='Blog Post Title']", :visible => false)
       expect(page).to have_css("meta[property='og:type'][content='article']", :visible => false)
-      expect(page).to have_css("meta[property='og:url'][content='/blog/blog-post-title']", :visible => false)
+      expect(page).to have_css("meta[property='og:url'][content='http://mydomain.com/blog/blog-post-title']", :visible => false)
       expect(page).to have_css("meta[property='og:description'][content='Blog post description']", :visible => false)
-      expect(page).to have_css("meta[property='og:image'][content='/uploads/blog_image/image/1/test.png']", :visible => false)
+      expect(page).to have_css("meta[property='og:image'][content='http://mydomain.com/uploads/blog_image/image/1/test.png']", :visible => false)
     end
 
     scenario 'should have twitter meta tags' do
-      expect(page).to have_css("meta[name='twitter:card'][content='summary']", :visible => false)
+      expect(page).to have_css("meta[name='twitter:card'][content='summary_large_image']", :visible => false)
```

Comment:
No. That's how it's required by twitter: https://dev.twitter.com/cards/types/summary-large-image

```html
<meta name="twitter:card" content="summary_large_image">
<meta name="twitter:site" content="@nytimes">
<meta name="twitter:creator" content="@SarahMaslinNir">
<meta name="twitter:title" content="Parade of Fans for Houston’s Funeral">
<meta name="twitter:description" content="NEWARK - The guest list and parade of limousines with celebrities emerging from them seemed more suited to a red carpet event in Hollywood or New York than than a gritty stretch of Sussex Avenue near the former site of the James M. Baxter Terrace public housing project here.">
<meta name="twitter:image:src" content="http://graphics8.nytimes.com/images/2012/02/19/us/19whitney-span/19whitney-span-articleLarge.jpg">
```